spec/routing/route_spec.rb

```diff
@@ -115,12 +115,6 @@
     end
   end
 
-  describe 'Authorities' do
```

Comment:
@jcoyne yes and no:  this was the line I was thinking needed a test: https://github.com/projecthydra/sufia/pull/2288/files#diff-25c1ec9f5b287e7f6a20f696507ab602R24
There may be a better test than a route test to make sure the engine was mounted.